core/src/main/java/com/xatkit/core/recognition/IntentRecognitionProviderFactory.java

```diff
@@ -125,7 +140,14 @@ private static RecognitionMonitor getRecognitionMonitor(XatkitCore xatkitCore,
          */
         RecognitionMonitor monitor = null;
         if (configuration.isEnableRecognitionAnalytics()) {
-            monitor = new RecognitionMonitor(xatkitCore.getXatkitServer(), configuration.getBaseConfiguration());
+            if(configuration.getBaseConfiguration().getString(XATKIT_DATABASE_KEY, DEFAULT_DATABASE).equals(INFLUX_DATABASE)){
+                Log.info("Starting with influxDB");
```

Comment:
I replaced this with "Using InfluxDB to store monitoring data". "Starting with influxDB" is not precise enough for the end user (what is InfluxDB used for?).
I did the same for the MapDB log